control-plane/api/v1alpha1/proxydefaults_types.go

```diff
@@ -75,6 +75,8 @@ type ProxyDefaultsSpec struct {
 	MeshGateway MeshGateway `json:"meshGateway,omitempty"`
 	// Expose controls the default expose path configuration for Envoy.
 	Expose Expose `json:"expose,omitempty"`
+	// AccessLogs controls all envoy instances' access logging configuration.
+	AccessLogs *AccessLogs `json:"accessLogs,omitempty"`
 }
 
 func (in *ProxyDefaults) GetObjectMeta() metav1.ObjectMeta {
```

Comment:
nit: public method should have comment (I know you didn't create this)

Reply:
Looks like the `ConfigEntryResource` interface has a comment for this. I don't see comments in other implementations, so I'll probably pass on updating this round.